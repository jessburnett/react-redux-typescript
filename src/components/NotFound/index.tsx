import * as React from 'react';

export const NotFound = (): JSX.Element => <h1>NOT FOUND</h1>;

export default NotFound;
