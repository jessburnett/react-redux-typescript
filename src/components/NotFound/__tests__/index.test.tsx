import * as React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../';

describe('<NotFound/>', () => {

    it('renders correctly', () => {
        expect(shallow(<NotFound/>)).toMatchSnapshot();
    });

});
