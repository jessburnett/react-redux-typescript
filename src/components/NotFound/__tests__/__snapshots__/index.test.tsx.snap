exports[`<NotFound/> renders correctly 1`] = `
<h1>
  NOT FOUND
</h1>
`;
